### DISCUSSION TITLE
Welcome to speedy-tuner Discussions!

### DISCUSSION THREAD

**Discussion by karniv00l** [hyper-tuner/hyper-tuner#133] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by max-d3v** [comment]
Hey!, i’ve been exploring car remapping and tuning software lately and came across your project. I’m really interested in learning more and possibly building something similar myself.

I was wondering if this project has been abandoned? If so, would you consider adding me as a watcher to the private repo so I could take a look and potentially contribute or even continue it in the future? No worries at all if that’s not possible—just thought I’d ask!
